WordPress/Sniffs/WP/EnqueuedCheckSniff.php

```diff
@@ -0,0 +1,119 @@
+<?php
```

Comment:
Looks like you've committed the same sniff twice with two different names. (Or maybe I am missing something?) 

Perhaps `EnqueuedCheck` is a better name than `EnqueuedVersion`, since the sniff also checks the `$in_footer` arg. 

Does anybody else have an opinion?

Reply:
@JDGrimes Thanks for checking the code :)

Yes I added this one in as `EnqueuedCheck` seemed better than `EnqueuedVersion` hence why I removed the `EnqueuedResourcesSniff.php ` file.  Sorry about that